common/src/main/kotlin/com/gitlab/kordlib/common/Color.kt

```diff
@@ -9,12 +9,12 @@ class Color(val rgb: Int) {
     val blue: Int get() = (rgb shr 0) and 0xFF
 
     init {
-        if (rgb < 0 || rgb > 0xFFFFFF) throw IllegalArgumentException("Color is invalid.")
+        if (rgb !in 0..0xFFFFFF) throw IllegalArgumentException("Color is invalid.")
     }
 }
 
-
 private fun rgb(red: Int, green: Int, blue: Int): Int {
+    if(red !in 0..255 || green !in 0..255 || blue !in 0..255) throw IllegalArgumentException()
```

Comment:
This should probably throw a more informative exception, inform what the expected vs actual value is, which specific value was illegal.